pvlib/horizon.py

```diff
@@ -0,0 +1,459 @@
+'''
+The 'horizon' module contains function definitions that
+retrive & calculate the surrounding horizon using DEM
+elevation data
+'''
```

Comment:
Add a note here listing the optional dependences that are required for use of this module.